src/main/scala/com/microsoft/hyperspace/actions/RefreshActionBase.scala

```diff
@@ -70,11 +70,19 @@ private[actions] abstract class RefreshActionBase(
     val latestRelation =
       Hyperspace.getContext(spark).sourceProviderManager.refreshRelation(relations.head)
     val dataSchema = DataType.fromJson(latestRelation.dataSchemaJson).asInstanceOf[StructType]
-    spark.read
-      .schema(dataSchema)
-      .format(latestRelation.fileFormat)
-      .options(latestRelation.options)
-      .load(latestRelation.rootPaths: _*)
+    if (latestRelation.rootPaths.size == 1) {
```

Comment:
Delta Lake only allows one path in load()

Reply:
Yea, we can do something like:
```scala
val df = spark.read
  .schema(dataSchema)
  .format(latestRelation.fileFormat)
  .options(latestRelation.options)
// Due to the difference in how the "path" option is set: https://github.com/apache/spark/blob/ef1441b56c5cab02335d8d2e4ff95cf7e9c9b9ca/sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala#L197,
// load() with a single parameter needs to be handled differently.
if (latestRelation.rootPaths.size == 1) {
  df.load(latestRelation.rootPaths.head)
} else {
  df.load(latestRelation.rootPaths: _*)
}
```

Reply:
Btw, what happens if the delta lake implementation adds the "path" option in `latestRelation.options`?

Reply:
https://github.com/microsoft/hyperspace/pull/265/files#diff-f7ecc68d1799e9c2c916973786081d5f35f312785537ccd2eed61bce41a10786R72
"path" key will be removed in `createRelation`.